myft/templates/save-for-later.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default ({ myFtApiWrite, contentId, variant, buttonText }) => {
+	return !myFtApiWrite ? null : (
+		<form className="n-myft-ui n-myft-ui--save" method="POST"
+				data-content-id={contentId}
+				data-myft-ui="saved"
+				action={`/__myft/api/core/saved/content/${contentId}?method=put`}>
+			<button
+					type="submit"
+					className={`n-myft-ui__button${variant ? 'n-myft-ui__button--' + variant : ''}`}
+					data-trackable="save-for-later"
+					data-alternate-label="Saved to myFT"
+					data-alternate-text="Saved"
+					data-content-id={contentId}
+					aria-label="Save to myFT"
+					title="Save to myFT"
+			>{buttonText || 'Save'}</button>
+		</form>
```

Comment:
I tend to like have 2 tabs for continuing lines, 1 tab for actual indention. If we have 1 tab for continuing lines, then it's hard to distinguish between continuing lines and actual indention. Interested to know your opinion on this.